index.js

```diff
@@ -2,66 +2,66 @@
   const canvas = document.getElementById("canvas");
   const context = canvas.getContext('2d');
   let gameSpeed = 30;
-  let computerPaddleSpeed = 5;
-  const MAX_SCORE = 3;
-  let playerScore = 0;
-  let computerScore = 0;
-
+  let paddleSpeed = 5;
 
+  class ObjXY {
+    constructor(x, y) {
+      this.x = x;
+      this.y = y;
+    }
+  }
 
   /**
-   * Ball Constructor
+   * Create instance of a ball
    */
-  function Ball() {
-    this.color = "white";
-    this.size = {
-      x: 10,
-      y: 10
+  class Ball {
+    constructor() {
```

Comment:
Could the constructor take setup arguments, and then you can merge game defaults into it. This would make it more flexible moving forward